test/e2e/service_test.go

```diff
@@ -133,7 +133,7 @@ func TestNodePortWindows(t *testing.T) {
 	_, err = data.podWaitForIPs(defaultTimeout, clientName, testNamespace)
 	require.NoError(t, err)
 
-	nodeIP := clusterInfo.nodes[0].ip
+	nodeIP := clusterInfo.nodes[0].ipv4Add
```

Comment:
Doesn't cause problem when the test is on IPv6?

Reply:
It seems all related tests has passed. This test case is in windows Node.